src/ui/layout/responsive/building-adaptive-apps.md

```diff
@@ -343,10 +341,18 @@ bool get isDesktopDeviceOrWeb => kIsWeb || isDesktopDevice;
 
 The `Platform` API can't be accessed from web builds without
 throwing an exception, because the `dart.io` package isn't
-supported on the web target. As a result, this code checks
+supported on the web target. As a result, the above code checks
 for web first, and because of short-circuiting,
 Dart never calls `Platform` on web targets.
 
+Use `Platform`/`kIsWeb` when the logic absolutely <i>must</i>
+run for a given platform. For example,
+talking to a plugin that only works on iOS,
+or displaying a widget that only conforms to
+Play Store policy and not App Store.
```

Comment:
Should this be something like?

```suggestion
Play Store policy and not the App Store's.
```

Reply:
Yes, it's like my English has taken vacation.